pages/forgot_password.php

```diff
@@ -0,0 +1,28 @@
+<?php
+setTitle("Wachtwoord vergeten");
+
+if ($_SERVER["REQUEST_METHOD"] == "POST") {
+    $forget = base_query("SELECT * FROM User WHERE Email=:email", array(
+            ":email" => $_POST["email"]
+    ))->fetch();
+    //var_dump($forget);
+    if ($_POST["email"] == ""){
```

Comment:
Er is een functie genaamd `empty` waarmee je kan kijken of iets leeg is. `if (empty($_POST['email'])) {`. Empty kijkt ook of de waarde misschien `null` is.